src/eventloop.jl

```diff
@@ -18,7 +18,15 @@ function eventloop(socket)
                 end
             finally
                 @async begin
-                    sleep(idle_delay[])
+                    if check_interact_loaded() && idle_delay[] < interact_min_delay
```

Comment:
I don't like specifically checking for Interact.   Would be better to patch Interact to use the same mechanism for controlling the idle delay, or simply to get rid of the idle delay entirely in favor of channels or whatever.